components/Dashboard/Dashboard.js

```diff
@@ -18,15 +25,99 @@ export default function Dashboard() {
     setSelectedColumn(column);
   }
 
+  const { allEntries, isLoadingEntries, errorEntries } =
+    useData().fetchedAllEntries;
+  const { userEntries } = useData().fetchedUserEntries;
+  const { data: session } = useSession();
+
+  const last7DaysEntries =
+    !isLoadingEntries && last7DaysAnalyser(session ? userEntries : allEntries);
+
+  const allEmotionsDisplayed = entriesToDisplay({
+    last7DaysEntries,
+    session,
+    isLast7Days,
+  });
+
+  const handleExperienceClick = (experience) => {
+    setClickedExperience(experience);
+    setTargetExperience(experience);
+    if (experience === targetExperience) {
+      setSingleExperienceList(!singleExperienceList);
+      setClickedExperience(null);
+    }
+  };
+
+  function entriesToDisplay({ last7DaysEntries, session }) {
```

Comment:
both variables are already accessible to this function without the need to pass it to the function. this would work too: 
`function entriesToDisplay() {`